docs/resources/Supported_Devices.md

```diff
@@ -0,0 +1,173 @@
+a remote development environment with access to Intel® hardware and the latest versions of the Intel® Distribution 
+of OpenVINO™ Toolkit. `Learn more <https://devcloud.intel.com/edge/get_started/devcloud/>`__ or `Register here <https://inteliot.force.com/DevcloudForEdge/s/>`__.
+
+
+Supported Configurations
+###########################################################
+
+The OpenVINO Runtime can inference models in different formats with various input and output formats.
+This page shows supported and optimal configurations for each plugin.
+
+
+Terminology
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+===================  =============================================
+ Acronym/Term         Description                                 
+===================  =============================================
+ FP32 precision       Single-precision floating-point format       
+ BF16 precision       Brain floating-point format                  
+ FP16 precision       Half-precision floating-point format         
+ I16 precision        2-byte signed integer format                 
+ I8 precision         1-byte signed integer format                 
+ U16 precision        2-byte unsigned integer format               
+ U8 precision         1-byte unsigned integer format               
+===================  =============================================
+
+NHWC, NCHW, and NCDHW refer to the data ordering in batches of images:
+
+* NHWC and NCHW refer to image data layout.
+* NCDHW refers to image sequence data layout.
+
+Abbreviations in the support tables are as follows:
+
+* N: Number of images in a batch
+* D: Depth. Depend on model it could be spatial or time dimension
+* H: Number of pixels in the vertical dimension
+* W: Number of pixels in the horizontal dimension
+* C: Number of channels
+
+CHW, NC, C  - Tensor memory layout.
+For example, the CHW value at index (c,h,w) is physically located at index `(c*H+h)*W+w`, for others by analogy.
+
+
+Supported Precision
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+==================  ===========================  ============================  ==========================
+ Plugin              FP32                         FP16                          I8                              
+==================  ===========================  ============================  ========================== 
+ CPU plugin          Supported and preferred      Supported                     Supported                                                        
+ GPU plugin          Supported                    Supported and preferred       Supported                                                        
+ GNA plugin          Supported                    Supported                     Not supported                                                    
+ Arm® CPU plugin     Supported and preferred      Supported                     Supported (partially)                                            
```

Comment:
we no longer have ARM plugin after merging it into CPU plugin, should we have it here? @dmitry-gorokhov, please advice

Reply:
I believe that we should not list it if it does not function as an independent plugin. But we should add information to the CPU plugin that it supports Arm. What do you think?